### PR TITLE
GA'ing v1 of service account tokens

### PR DESCRIPTION
**Pull request by drewbanin**
## Description & motivation
Added additional docs for service account API tokens. Thanks for the great start on this one @amychen1776 - just adding some more details as we get ready for GA D: 

## To-do before merge
- @iktl you know what you must do

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [x] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

### CODE REVIEW

**Comment by drewbanin** [comment]
@aaronraff wondering if you could give this one a gut-check after GAing service account tokens?

File: website/docs/docs/dbt-cloud/cloud-api.md [hunk 1]
```diff
@@ -3,30 +3,56 @@ title: "dbt Cloud API"
...
 - Download artifacts after a job has completed 
 - Kick off a job run from an orchestration tool
 - Manage your dbt Cloud account
+- and more
+
+Full reference documentation for the dbt Cloud API can be found [here](/dbt-cloud/api).
+
+## How to Access the API 
+
+dbt Cloud support two types of API Tokens: User tokens and Service Account
```
Comment (aaronraff) [inline website/docs/docs/dbt-cloud/cloud-api.md]: Should this be "supports"?

**Review by aaronraff — APPROVED** [review]
I left one incredibly nit-picky (and optional) suggestion.

This looks great to me, thanks for putting this together 😄

File: website/docs/docs/dbt-cloud/cloud-api.md (hunk 1 above)
Comment (drewbanin) [reply website/docs/docs/dbt-cloud/cloud-api.md]: ...and that's why we do code review! Thanks @aaronraff - merging :)

**Pull request merged**